rollbot/profile/profile.py

```
@@ -0,0 +1,101 @@
+# profile.py
+
+import re
+import sqlite3
+import aiosqlite
+import rollbot.profile.profile
+
+
+class Profile():
+    def __init__(self, db_name: str, table_suffix: str):
+        self.db_name = db_name
+        self.table = f'PROFILES{table_suffix}'
+        self.create_db()
+
+    def create_db(self):
+        try:
+            with sqlite3.connect(self.db_name) as db:
+                db.execute(f'''CREATE TABLE {self.table}
+                    (ID      INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+                    OWNER    CHAR(50)                          ,
+                    OWNER_ID INTEGER                           NOT NULL,
+                    PROFILE  CHAR(50)                          NOT NULL,
+                    LINK     CHAR(255)                         NOT NULL);''')
```

We're gently lying here if we define something as CHAR(num), as sqlite will just silently convert it to type TEXT for us. I'd prefer it if we just use TEXT, as that matches implementation.

---

Repaired.